index.html

```diff
@@ -1,252 +1,248 @@
+                    </li>
+                    <li>
+                        <a href="#">
                             <img src="img/Vector4.png" alt="github picture">
-                        </li>
-                    </ul>
-
-                    <ul>
-                        <li>
-                            <a href="#"></a>
-                            <img src="img/Vector5.png" alt=" peace sign picture" >
-                        </li>
-                    </ul>
-                 </div>
-        </section>
-
-        <section class="canopy-card">
-            <section class="canopy-card-section">
-                <div class="snapshoot-portofolio">
-                    <img src="img/Snapshoot Portfolio.png" alt="snapshoot Portfolio picture">
-                </div>
-                <div class="project-card">
-                    <h2>Tonic</h2>
-                    <h3>CANOPY &nbsp; <img src="img/Counter.png" alt="dot symbol">&nbsp;<span> Back End Dev &nbsp; <img src="img/Counter.png" alt="dot symbol"> &nbsp; 2015 </span></h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul>
-                        <li class="html-tag">html</li>
-                        <li class="css-tag">css</li>
-                        <li class="js-tag">javascript</li>
-                    </ul>
-                    <button type="button">See Project</button>
-                </div>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="img/Vector5.png" alt=" peace sign picture">
+                        </a>
+                    </li>
+                </ul>
             </section>
+        </main>
+        <section class="canopy-card">
 
-            <section class="canopy-second-card-section">
-                <div class="snapshoot-portofolio">
-                    <img src="img/Snapshoot Portfolio (1).png"alt="snapshoot-portofolio">
-                </div>
-                <div class="project-card">
-                    <h2>Multi-Post <br />Stories</h2>
-                    <h3>CANOPY &nbsp;<img src="img/Counter.png" alt="dot symbol"> &nbsp; <span> Back End Dev &nbsp;<img src="img/Counter.png"alt="dot symbol"> &nbsp; 2015</span></h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul>
-                        <li>html</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button type="button" disabled>See Project</button>
+            <div class="canopy-card-section">
+                <div class="mycanopy-section">
+                    <div class="snapshoot-portofolio">
+                        <img src="img/Snapshoot Portfolio(10).png" alt="snapshoot Portfolio picture">
+                    </div>
```

Comment:
- You did a great job in your design but kindly make sure you use the appropriate path to display your image on my side the image are broken please fix that  :+1: